common/include/pcl/conversions.h

```diff
@@ -301,7 +306,8 @@ namespace pcl
       for (std::size_t x = 0; x < cloud.width; x++)
       {
         std::uint8_t * pixel = &(msg.data[y * msg.step + x * 3]);
-        memcpy (pixel, &cloud (x, y).rgb, 3 * sizeof(std::uint8_t));
+        auto rgb{cloud(x, y)};
+        std::copy_n(&rgb, 3, pixel);
```

Comment:
Was `&cloud(x,y).rgb` proving problematic?

Reply:
I think I was trying to find out its type, but nvm, I'll restore it